src/Infrastructure/Controller/Regulation/Fragments/GetLocationController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller\Regulation\Fragments;
+
+use App\Application\Regulation\View\Measure\MeasureView;
+use App\Domain\Regulation\Repository\LocationRepositoryInterface;
+use App\Domain\Regulation\Repository\MeasureRepositoryInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\Requirement\Requirement;
+
+final class GetLocationController extends AbstractController
+{
+    public function __construct(
+        private \Twig\Environment $twig,
+        private LocationRepositoryInterface $locationRepository,
+        private MeasureRepositoryInterface $measureRepository,
+    ) {
+    }
+
+    #[Route(
+        '/_location/{uuid}',
+        name: 'get_location',
+        requirements: ['uuid' => Requirement::UUID],
+        methods: ['GET'],
+    )]
+    public function __invoke(Request $request, string $uuid = ''): Response
```

Comment:
```suggestion
    public function __invoke(Request $request, string $uuid): Response
```

Le paramètre UUID est en requirement, il n'est donc pas possible qu'il soit null.

Reply:
En fait, j'ai besoin de la valeur par défaut `$uuid = ''` pour récupérer le chemin  de ce contrôleur dans `map.html.twig` avec un `{{ path('fragment_map_location') }}`